src/modules/CourseDetail/index.tsx

```diff
@@ -37,10 +38,10 @@ export function CourseDetailPage(props: { data: GetCourseResponse }) {
   const SEOConfig: NextSeoProps = {
     ...defaultSEO,
     title: course.abbrName,
-    description: course.courseDesc ?? defaultSEO.description,
+    description: course.courseDescTh + '\n' + course.courseDescEn ?? defaultSEO.description,
```

Comment:
Maybe try something like this

```suggestion
    description: [course.courseDescTh, course.courseDescEn].filter(desc => !!desc).join('\n') || defaultSEO.description,
```